homeassistant/components/unifi/entity.py

```diff
@@ -92,6 +92,12 @@ def async_client_device_info_fn(api: aiounifi.Controller, obj_id: str) -> Device
     )
 
 
+@callback
+def async_device_outlet_supported_fn(controller: UniFiController, obj_id: str) -> bool:
```

Comment:
```suggestion
def async_device_outlet_power_supported_fn(controller: UniFiController, obj_id: str) -> bool:
```
This should be in the sensor.py file as it's only relevant for the sensor right now. Should we use the cap for this one? Even though we don't have all data I think that's the way to go. What do you think?

Reply:
let's give it a shot!